Azure/azure-cli
# [AMS] Fix #14021: az ams account sp is not idempotent

**Description**  
Fix for #14021 

**Testing Guide**  
Running the following command multiple times should output the same SP object
`az ams sp create -n [sp-name] -a [account-name] -g [resource-group]`

To change the name or role of an existing SP run the following command:
`az ams sp create -n [sp-name] --new-sp-name [new-sp-name] -a [account-name] -g [resource-group] --role [role]`


**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
